packages/react-components/src/components/MessageThread.tsx

```diff
@@ -1275,8 +1209,6 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
       messages,
       showMessageDate,
       showMessageStatus,
-      onRenderAvatar,
-      isNarrow,
```

Comment:
The isNarrow adjustments of the avatar overlap have been lost; need to get this back into the rendering method of the chatMessage from f9